packages/wdio-runner/src/index.js

```diff
@@ -153,12 +154,14 @@ export default class Runner extends EventEmitter {
      * init WebDriver session
      * @param  {object}  config        configuration of sessions
      * @param  {Object}  caps          desired cabilities of session
+     * @param  {Object}  testData      Test data for the instance
      * @return {Promise}               resolves with browser object or null if session couldn't get established
      */
-    async _initSession (config, caps) {
+    async _initSession (config, caps, testData) {
         let browser = null
 
         try {
+            global.testData = testData
```

Comment:
I am not sure if `testData` is the best variable name for this

Reply:
@christian-bromann : Could you please suggest us some names. Does ```data``` sounds good?